R/process_sc_all_care_home.R

```diff
@@ -28,53 +29,63 @@ process_sc_all_care_home <- function(
     spd_path = read_file(get_spd_path()),
```

Comment:
on line 28 `ch_name_lookup_path` the `read_file` is not working due to the document being xlsx

Reply:
```
readxl::read_excel(get_slf_ch_name_lookup_path())
```
This seems to work in place

Reply:
fixed